.github/renos_updated.sh

```diff
@@ -11,7 +11,8 @@ if [ -z $CHANGED_SOURCE_FILES ]; then
     exit 0;
 fi;
 
-for file in $CHANGED_SOURCE_FILES
+CHANGED_RELEASE_NOTES=$(git diff --name-only origin/main $GITHUB_SHA -- releasenotes)
+for file in $CHANGED_RELEASE_NOTES
```

Comment:
In a previous [PR](https://github.com/openqasm/openqasm/pull/511), I broke this script. `CHANGED_SOURCE_FILES` contains only changes to the `source` directory, so no notes would be detected. The above should fix this.

Reply:
Easily enough done.  I guess at some point we might need to look into writing a manual "I know what I'm doing, skip this check" label or something, but definitely not in this PR.